src/web/user_agent.rs

```diff
@@ -58,6 +58,12 @@ impl DeviceInfo {
         self.os_family == OsFamily::IOs && self.ff_version < 36
             || self.os_family != OsFamily::IOs && self.ff_version < 91
     }
+
+    /// Determine if the device is a mobile phone based on either the form factor or OS.
+    pub fn is_mobile(&self) -> bool {
+        self.form_factor == FormFactor::Phone
```

Comment:
@jrconlin Just confirmed with the mobile team, we do have mobile traffic on "tablets", and they wanted Contile to respond to tile requests from there. So let's include `FormFactor::Tablet` here as well.

Reply:
Should we run this by adM as well?

Reply:
Yeah, we will confirm this with them in our next meeting.